include/sys/cbprintf_internal.h

```diff
@@ -127,9 +127,9 @@ extern "C" {
 #define Z_CBPRINTF_MUST_RUNTIME_PACKAGE(skip, ...) ({\
 	_Pragma("GCC diagnostic push") \
 	_Pragma("GCC diagnostic ignored \"-Wpointer-arith\"") \
-	int _rv = COND_CODE_0(NUM_VA_ARGS_LESS_1(__VA_ARGS__), \
-			(0), \
-			(((Z_CBPRINTF_HAS_PCHAR_ARGS(__VA_ARGS__) - skip) > 0))); \
+	bool _rv = COND_CODE_0(NUM_VA_ARGS_LESS_1(__VA_ARGS__), \
+			(false), \
```

Comment:
Is this the same coding guideline? Maybe this should be a separate commit.

Reply:
I'll fix that.